index.html

```diff
@@ -170,11 +170,18 @@
         draw.rect(tree.width, tree.height / 4).attr({fill: '#4d3617', opacity:"0.3"}).click(hideHelp).y(row_height * 2);
 
         // Add Age Icons
-        let icon_height = 112;
-        let vertical_spacing = (row_height - icon_height) / 2;
+        let icon_height = 112, icon_width = 112;
+        let vertical_spacing = (row_height - icon_height) / 2 - 10;
+        let margin_left = 20;
         let image_urls = ['dark_age_de.png', 'feudal_age_de.png', 'castle_age_de.png', 'imperial_age_de.png'];
+        let age_texts = ['Dark Age', 'Feudal Age', 'Castle Age', 'Imperial Age'];
```

Comment:
We should not hard code those, but take them from the language data (see also https://github.com/SiegeEngineers/aoe2techtree/pull/33)